### DISCUSSION TITLE
Integrating file sharing with ChatStatefulClient

### DISCUSSION THREAD

**Discussion by AlbertoPerez8** [Azure/communication-ui-library#3463] [Q&A]
Is there any documentation or outlined steps on adding file sharing functionality to a chat app that runs with Chat Clients Provider components? Or is is better to implement a custom message case and use the communications SDK and sending the message independently from the chat client provider component?

**Comment by jimchou-dev** [comment]
Please see response in https://github.com/Azure/communication-ui-library/issues/3505
